R/par.R

```diff
@@ -185,10 +190,16 @@ igraph_options <- function(...) {
 
   ## Set them
   names(temp) <- paste0("igraph::", names(temp))
-  do.call(set_config_in, c(temp, list(.in = parent.frame())))
+  do.call(set_config_in, c(temp, list(.in = .in)))
   invisible(old)
 }
 
+local_igraph_options <- function(..., .in = parent.frame()) {
+  old <- igraph_options(..., .in = .in)
+  withr::defer(rlang::inject(igraph_options(!!!old)), envir = .in)
```

Comment:
I guess that we need to move `withr` from the `Suggests:` section of the `DESCRIPTION` file to the `Imports:` section, right?

Reply:
Not necessarily. This isn't part of an exported API, and we can realistically expect that it's available during testing.